### DISCUSSION TITLE
一些库已失效

### DISCUSSION THREAD

**Discussion by XingShuyu** [Tsuk1ko/CQ-picfinder-robot#231] [问题求助] [answered]
部署时出现这样的提示，正常吗？
![Screenshot_20210724_180056](https://user-images.githubusercontent.com/82715884/126865084-fff638d2-7ca4-420e-afb3-c35b52558fce.jpg)

**Comment by Miuzarte** [comment, marked as answer]
找台正经的Linux你就会发现没这问题了

**Comment by Miuzarte** [comment]
对了  你可以试试用docker
[[KotoriK/cqpicfinder-docker]](https://github.com/KotoriK/cqpicfinder-docker)
